app/controllers/api/functions.php

```diff
@@ -1381,6 +1750,8 @@
             throw new Exception(Exception::VARIABLE_ALREADY_EXISTS);
         }
 
+        $dbForProject->updateDocument('functions', $function->getId(), $function->setAttribute('live', false));
+
         $schedule = $dbForConsole->getDocument('schedules', $function->getAttribute('scheduleId'));
```

Comment:
Let's add a small comment to explain what we do every time we update the schedule document.

Reply:
Added comments